### DISCUSSION TITLE
testing: CI Tests for new releases of Node API

### DISCUSSION THREAD

**Discussion by YazzyYaz** [celestiaorg/celestia-node#3027] [Ideas]
### Implementation ideas

We need to have CI tests for testing Node API new releases to ensure the API works as expected. We can use Prompt Scavenger tutorial as well to test that it works as expected or otherwise we would need to update the docs if something changed in the Node API logic or resolve Node API bugs if things break that make the tutorial not work anymore.

https://docs.celestia.org/developers/prompt-scavenger/

cc @Bidon15 @sysrex @renaynay @Wondertan @distractedm1nd

**Comment by Wondertan** [comment]
Noting that the Go libs are part of Celestia-node repo CI testing. Js, Rust, and others will need something else

**Comment by renaynay** [comment]
Our swamp tests will now use RPC instead of the node directly, but I still think it makes sense to have a tutorial or something like that pointed at **at least** the blob module to catch breaks.
